### ISSUE TITLE
How to get memory diff between first and second batch?

### ISSUE THREAD

**Issue by khursani8** [Stonesjtu/pytorch_memlab#48]
I'm using MemReporter
After run `reporter.report()`
I don't know which part different since there a lot of layer and tensor being output
Wondering if there is a way to find diff easily

**Comment by Stonesjtu** [comment]
Good idea, I'm trying to add a special symbol (like *) along the tensor name for newly added tensors.

Stay tuned.
